apps/swirl-docs/src/components/IconGrid/IconGridItem.tsx

```diff
@@ -0,0 +1,41 @@
+import { FlipPopover } from "@getflip/swirl-components-react";
+import { FunctionComponent } from "react";
+import { IconsMetaData } from "src/pages/icons";
+import IconInfo from "./IconInfo";
+
+interface IconGridProps {
+  id: string;
+  icon: string;
+  icons: IconsMetaData;
+  handleTileClick: (iconName: string) => void;
+}
+
+const IconGridItem: FunctionComponent<IconGridProps> = ({
+  id,
+  icon,
+  icons,
+  handleTileClick,
+}) => {
+  return (
+    <li>
+      <a
+        id={id}
+        href={`#${icons[icon]?.name}`}
+        onClick={() => handleTileClick(icons[icon]?.name)}
+        className="flex flex-col justify-center items-center py-4 border-1 rounded-lg"
+      >
+        <i
+          className={`swirl-icons-${icons[icon]?.name}28 text-icon-strong`}
+        ></i>
+        <span className="text-text-subdued">{icons[icon]?.name}</span>
+      </a>
+      <FlipPopover label="Icon Info" popoverId={`popover-${id}`} trigger={id}>
```

Comment:
**issue:** Only the icon and label are clickable right now. The bottom sheet doesn't open if I click the white space of the cell. Maybe you could set the `a` element to 100% height and set the `<li>` as trigger.

Reply:
good point. did not notice that